lib/ws-kernel-picker.coffee

```diff
@@ -0,0 +1,117 @@
+{SelectListView} = require 'atom-space-pen-views'
+_ = require 'lodash'
+
+Config = require './config'
+services = require './jupyter-js-services-shim'
+WSKernel = require './ws-kernel'
+uuid = require 'uuid'
+
+class CustomListView extends SelectListView
+    initialize: (@emptyMessage, @onConfirmed) ->
+        super
+        @addClass('watch-language-picker')
+        @storeFocusedElement()
```

Comment:
Restoring the focused pane isn't working for me. I think it has something to do with the location where `@storeFocusedElement()` is called.